tests/test_manager.py

```diff
@@ -138,57 +138,47 @@ def test_save_to_folders(tmpdir, manager, result, parameter_combination_range):
                             str(tmpdir.join('folder_export')),
                             2)
 
-def test_run_logging_simulation(ns_3_compiled_debug,config):        #TODO cannot find the scripts in /examples folder(even if --enable-examples is passed in the ns_3_compiled_debug fixture in conftest.py). Why does this happen?
-    params = {
-        'powerLevels': 18.0,
-        'manager': 'ns3::ParfWifiManager',
-        'AP1_y': 0,
-        'STA1_y': 0,
-        'maxPower': 17,
-        'outputFileName': 'parf',
-        'AP1_x': 0,
-        'STA1_x': 5.0,
-        'stepsTime': 1,
-        'minPower': 0,
-        'steps':200,
-        'rtsThreshold': 2346,
-        'stepsSize':1
-    }
+def test_run_logging_simulation(ns_3_compiled_debug,config):        
     log_component = {
-        'PowerAdaptationDistance':'debug'
+        'Logger':'all'
     }
+    # This should raise an exception as the campaign in built in optimized mode and logging is enabled                                            
     new_campaign = sem.CampaignManager.new(ns_3_compiled_debug,
-                                            'wifi-power-adaptation-distance',
+                                            'logging-ns3-script',
                                             config['campaign_dir'],
                                             overwrite=True)
 
-    #Ensure that the simulations run fine                                            
-    log_path = new_campaign.run_missing_simulations(params,log_component=log_component)                                        
+    with pytest.raises(Exception):
```

Comment:
Exception is too generic, tests should be more specific. if the code raises a generic Exception, it should be updated.
check [here](https://docs.python.org/3/library/exceptions.html#Exception)